app/screens/conversion-flow/conversion-confirmation-screen.tsx

```diff
@@ -118,14 +123,18 @@ export const ConversionConfirmationScreen = ({
           },
         })
 
-        const status = data.intraLedgerPaymentSend.status
+        const status = data?.intraLedgerPaymentSend.status
+
+        if (!status) {
+          throw new Error("Conversion failed")
```

Comment:
Is this caught somewhere or does it show the error screen?

Reply:
Caught in line 171